TESTS/spm/client_tests/server.c

```diff
@@ -34,16 +33,17 @@ void server_main(void *ptr)
                     break;
                 }
                 case PSA_IPC_MSG_TYPE_CALL: {
-                    uint8_t expect = 0;
-                    uint8_t off = 0;
-                    SPM_ASSERT(msg.response_size >= expect);
-                    if (msg.size > 1) {
-                        psa_read(msg.handle, 0, &expect, 1);
-                        psa_read(msg.handle, 1, &off, 1);
-                        psa_read(msg.handle, off, msg_buf, msg.size - off);
+                    memset(data, 0, sizeof(data));
+                    if (msg.size[0] + msg.size[1] + msg.size[2] > 1) {
```

Comment:
Why is this now reading three bytes per cycle instead of just one?

Reply:
his is part of the alignment with the PSA-IPC spec ARMmbed/PSA-IPC-doc#104

Reply:
The specification change exposes arrayof  input buffers to the secure function reader instead of a single concatenated buffer.
The change modifies the testing logic to reflect such a change.